src/adminDashboard/adminDashboard.ts

```diff
@@ -51,14 +57,100 @@ export class AdminDashboard {
   async hydrate(): Promise<void> {
     if (this.ethereumService.defaultAccountAddress) {
       const defaultAccount = this.ethereumService.defaultAccountAddress.toLowerCase();
-      this.seeds = this.seedService.seedsArray
-        .filter((seed) => { return seed.admin.toLowerCase() === defaultAccount;});
+      this.seeds = this.seedService.seedsArray;
     } else {
       this.seeds = [];
     }
   }
 
-  connect():void {
+  async setSeed(index) {
+    this.selectedSeed = this.seeds[index];
+  }
+
+  async fundSeed() {
+    try {
+      await this.selectedSeed.projectTokenContract.transfer(this.selectedSeed.address, this.selectedSeed.seedAmountRequired);
```

Comment:
You should be using the TransactionsService for sending any transaction.

Reply:
okay

Reply:
done